libs/partners/databricks/langchain_databricks/embeddings.py

```diff
@@ -0,0 +1,91 @@
+from typing import Any, Dict, Iterator, List
+
+from langchain_core.embeddings import Embeddings
+from langchain_core.pydantic_v1 import BaseModel, PrivateAttr
+
+from langchain_databricks.utils import get_deployment_client
+
+
+class DatabricksEmbeddings(Embeddings, BaseModel):
+    """Databricks embedding model integration.
+
+    Setup:
+        Install ``langchain-databricks``.
+
+        .. code-block:: bash
+
+            pip install -U langchain-databricks
+
+        If you are outside Databricks, set the Databricks workspace
+        hostname and personal access token to environment variables:
+
+        .. code-block:: bash
+
+            export DATABRICKS_HOSTNAME="https://your-databricks-workspace"
+            export DATABRICKS_TOKEN="your-personal-access-token"
+
+    Key init args — completion params:
+        endpoint: str
+            Name of Databricks Model Serving endpoint to query.
+        target_uri: str
+            The target URI to use. Defaults to ``databricks``.
+        query_params: Dict[str, str]
+            The parameters to use for queries.
+        documents_params: Dict[str, str]
+            The parameters to use for documents.
+
+    Instantiate:
+        .. code-block:: python
+            from langchain_databricks import DatabricksEmbeddings
+            embed = DatabricksEmbeddings(
+                endpoint="databricks-bge-large-en",
+            )
+
+    Embed single text:
+        .. code-block:: python
+            input_text = "The meaning of life is 42"
+            embed.embed_query(input_text)
+
+        .. code-block:: python
+            [
+                0.01605224609375,
+                -0.0298309326171875,
+                ...
+            ]
+
+    """
+
+    endpoint: str
+    """The endpoint to use."""
+    target_uri: str = "databricks"
+    """The parameters to use for queries."""
+    query_params: Dict[str, str] = {}
```

Comment:
are there any non-string params?

Reply:
Great catch, yes, there seems to be some non-string params. 
https://platform.openai.com/docs/api-reference/embeddings/create